indexclient/client.py

```diff
@@ -192,6 +192,61 @@ def create_alias(
         resp = self._put(url, headers=headers, data=data, auth=self.auth)
         return resp.json()
 
+    def get_latest_revision(self, did):
+        # type: (str) -> Document | None
+        doc = self._get("index", did, "latest").json()
+
+        if doc and "did" in doc:
+            return Document(self, doc["did"], doc)
+        return None
+
+    def auto_add_revision(self, did, version_incr_func=None):
```

Comment:
Will do nothing if `version_incr_func` is not passed,
Better make += 1 to version by default?

Reply:
:+1:  for standardization